### ISSUE TITLE
Its urgent - We need a Daedalus Version with low ram usage

### ISSUE THREAD

**Issue by gitmachtl** [input-output-hk/daedalus#3181]
### solution needed for people to migrate off from daedalus
its nice that we have all the new governance stuff, but people are suffering from the lack of support for daedalus for a long time.

many of those can't easily upgrade there systems to 32GB+ of ram. many are not technical experienced to do that on there own if even possible with there current system.

many don't have there seed phrase anymore, but could send off there funds to a new wallet. but not if they can't start daedalus anymore.

i have to deal with such people every day. i am an SPO, and we are many times the first ones who gets contacted by frustated users.

i know that daedalus is currently that child that noone wants, but its bad for the ada holders out there tbh. 

### idea
so an idea came up to at least release a daedalus version that is using an external node for the sync, so people need less ram.  cardano-wallet uses the rest.

doable?

at least they could then send of there funds to LACE, ETERNL, etc.

❗  we need to react, now. its getting worse every day we wait. 

-- Martin

**Comment by CryptoBlocks-pro** [comment]
Yeah, as an SPO as well, this comes up for our team as well. Great idea. Eric

**Comment by btbf** [comment]
This is a very good point!
Even in the Japanese community, many people use Daedalus, but there are also many people who don't use dAapps, and we often get inquiries about problems with computer specs. This needs to be resolved immediately.

**Comment by onyxstakepool** [comment]
[Issue #3182](https://github.com/input-output-hk/daedalus/issues/3182) provides a quick fix for Daedalus on MacOS.

**Comment by OYSTERPOOL** [comment]
I'm also regularly trying to help community members with Daedalus sync issues. A toggle switch to enable connection to an external node would be great to enable people to send their funds to a new wallet in the cases of lost seed phrase.

**Comment by kevinhammond** [comment]
We do have a version of the node in preparation that will limit RAM usage by using out-of-memory storage.  It's not quite production ready - we're looking for some people to help test it on a DevNet.

**Comment by kevinhammond** [comment]
> I'm also regularly trying to help community members with Daedalus sync issues. A toggle switch to enable connection to an external node would be great to enable people to send their funds to a new wallet in the cases of lost seed phrase.

I'll pass this on

**Comment by thevictor13** [comment]
If I recall correctly, back in the day newer Daedalus versions could import wallets from the previous ones. As an alternative solution to inaccessible wallets due to the RAM limitation it would be nice to have a tool to extract those wallets to import them in a light wallet?

**Comment by vhulchenko-iohk** [comment]
Hi everyone,

Thank you for reporting this issue and providing additional context.

I'm Volo, a Product Manager at IOG, responsible for maintaining the Daedalus wallet.

I'm currently learning about Daedalus wallet users, their issues, pain points, and needs. If you are a Daedalus wallet user or have any insights to share, I would love to speak with you. Feel free to set up a 30-minute call at your convenience using my calendar: https://calendly.com/volodymyr-hulchenko-iohk/30min

Looking forward to talking with you.

Best regards,
Volo

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
